data/Ajiboso-Adeola.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Ajiboso Adeola",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Enthusiast | Frontend Developer | Web3 Enthusiast |Writer",
+  "tags": ["Open Source", "Javascript", "Node.js"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/dde_olla" },
+    { "platform": "github", "url": "https://github.com/Ajiboso-Adeola" },
+    { "platform": "Hashnode", "url": "https://hashnode.com/@Deola" }
+  ],
+  "avatar": "https://github.com/Ajiboso-Adeola.png",
+  "links": [
+    {
+      "name": "Connect with me on Twitter",
+      "url": "https://twitter.com/dde_olla",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on github",
+      "url": "https://github.com/Ajiboso-Adeola",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/adeola-ajiboso-a6842a183",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Open Source Blog",
+      "url": "https://hashnode.com/@Deola",
+      "icon": "hashnode"
+    }
+   
```

Comment:
I think the CI is failing due to this extra space

Reply:
Changes made